controller/src/test/java/io/pravega/controller/auth/AuthFileUtils.java

```diff
@@ -23,4 +28,26 @@ public static String credentialsAndAclAsString(String username, String password,
         // This will return a string that looks like this:"<username>:<pasword>:acl\n"
         return String.format("%s:%s:%s%n", username, password, acl);
     }
+
+    public static String createAclString(@NonNull List<String> aceStrings) {
+        StringBuilder builder = new StringBuilder();
+        for (String aceString: aceStrings) {
+            builder.append(aceString).append(";");
+        }
+        return builder.toString();
+    }
+
+    @SneakyThrows
+    public static void addAuthFileEntry(@NonNull FileWriter writer, @NonNull String entryAsString) {
+        StringBuilder builder = new StringBuilder(entryAsString);
+        writer.write(builder.append("\n").toString());
+    }
+
+    @SneakyThrows
+    public static void addAuthFileEntry(@NonNull FileWriter writer, @NonNull String username,
+                                           @NonNull String hashedPwd, @NonNull List<String> aceStrings) {
+        StringBuilder builder = new StringBuilder();
```

Comment:
Recommend using `String.format`. Easier to read.

Reply:
Sure, I've modified it and it looks better.